htdocs/core/class/commonobject.class.php

```diff
@@ -4256,6 +4256,10 @@ public static function getAllItemsLinkedByObjectID($fk_object_where, $field_sele
 			return -1;
 		}
 
+		if (!preg_match('/^[_a-zA-Z0-9]+$/', $field_select)) {
+			error_log("ERROR: Invalid value '$field_select' for parameter \$field_select in call to getAllItemsLinkedByObjectID(). Must be a single field name.");
```

Comment:
error_log must not be used.
You can use instead 
dol_syslog(message, LOG_ERR);